html-tables/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table id="StudentGradeTable">
```

Comment:
Common naming convention for `id`'s are to be `kebab-cased`. So this should be `id="student-grade-table` or `camel-cased`, so `id="studentGradeTable`